src/css/managerPage.css

```diff
@@ -0,0 +1,144 @@
+
+table {
+  font-family: arial, sans-serif;
+  border-collapse: collapse;
+  max-width:100%;
+  
+}
+
+td, th {
+  border: 1px solid #8f8f8f;
+  text-align: left;
+  padding: 5px;
+}
+
+tr:nth-child(even) {
+  background-color: #c3daca;
+}
```

Comment:
The entire table should follow the same color scheme as the rest of the website imo, and also not be transparent the odd rows.